README.md

```diff
@@ -1 +1,80 @@
-# Hello_Microverse_Project
\ No newline at end of file
+# Hello_Microverse_Project
+
+This project is learning setup for microverse student on how
+
+- Create branches with different base branches.
+- Follow Gitflow.
+- Understand the concept of version control.
+- Open pull requests in GitHub.
+- Write descriptive, easy to understand commit messages.
+- Write short, detailed, easy to understand descriptions of pull requests.
+- Write a descriptive and easy to understand README in business English for every project submitted.
+- Demonstrate an ability to submit a project for code review.
```

Comment:
- [ ] **[OPTIONAL]** In following with the microverse README template, this should be a paragraph statement, not itemized list. For example, something along the lines of the following should do:

**A 'Hello World' project to demonstrate understanding of some tools and concepts like git, version control, pull requests, commits, project submission**